release_notes.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python
+
+"""
+Generates a release notes markdown file with list of issues grouped by
+specified issue labels for a particular milestone in GitHub repo
+
+Tested on: Ubuntu 16.10, Windows 10, OS X El Capitan
+
+Usage:
+1. Create a file for config named 'local_release_notes.cfg' (sample below)
+
+SAMPLE OF CONFIG FILE:
+-----------------------------------------------------------------------
+[config]
+# the project repo path on GitHub
+repo = akvo/akvo-flow-mobile
+
+# the milestone for which the release notes are to be generated
+milestone = 2.2.11
+
+# the GitHub issue labels by which to filter and group the notes
+labels = New and noteworthy, Resolved issues
```

Comment:
it has to be labels = New and noteworthy,Resolved issues without the space otherwise it will not work